apps/web/src/utils/usernames.ts

```diff
@@ -419,7 +419,7 @@ export function normalizeName(name: string) {
 }
 
 // Assume domainless name to .base.eth
-export async function formatDefaultUsername(username: string | BaseName) {
+export async function formatDefaultUsername(username: string) {
```

Comment:
not worth putting `| Basename` here?

Reply:
Basename is a string, so it's redundant to add both,  string | string

Reply:
ah, i thought Basename had additional validation. makes sense